config/config.go

```diff
@@ -277,6 +277,11 @@ func FindConfigFilesInPath(rootPath string, terragruntOptions *options.Terragrun
 			return err
 		}
 
+		// Skip the Terragrunt cache dir entirely
+		if info.IsDir() && info.Name() == options.TerragruntCacheDir {
```

Comment:
Does `.Name()` return the folder name? Or the entire folder path? Because an equality check would only work for the former.

Reply:
Just the folder name.